release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -183,6 +183,11 @@ For more information, see xref:../installing/installing_aws/installing-restricte
 ==== Additional customization when installing {product-title}
 {product-title} 4.11 allows you to disable the installation of the `baremetal` and `marketplace` Operators, and the `openshift-samples` content that is stored in the `openshift` namespace. You can disable these features by adding the `baselineCapabilitySet` and `additionalEnabledCapabilities` parameters to the `install-config.yaml` configuration file prior to installation. If you disable any of these capabilities during the installation, you can enable them after the cluster is installed. After a capability has been enabled, it cannot be disabled again.
 
+[NOTE]
+====
+The `marketplace` Operator populates OperatorHub with a pre-defined xref:../operators/understanding/olm-rh-catalogs.adoc#olm-rh-catalogs_olm-rh-catalogs[set of Operator catalogs]. If you disable this Operator, OperatorHub will not be populated with these catalogs. You can xref:../operators/admin/olm-managing-custom-catalogs.adoc#olm-creating-catalog-from-index_olm-managing-custom-catalogs[add a custom catalog source to a cluster] if you want to install Operators after disabling the `marketplace` Operator. {product-title} does not automatically update manually created catalog sources.
```

Comment:
Can you try rewording this to avoid future tense? e.g. - `If you disable this Operator, these catalogs are not populated in OperatorHub` or `The ``marketplace`` Operator populates OperatorHub with a pre-defined xref:../operators/understanding/olm-rh-catalogs.adoc#olm-rh-catalogs_olm-rh-catalogs[set of Operator catalogs] only when the Operator is enabled` if it conveys the right message (minimalism).

Reply:
Reworded as "If you disable this Operator, these catalogs are not present in OperatorHub." WDYT?

Reply:
sounds better!